modules/virt-deploying-ssp.adoc

```diff
@@ -33,7 +33,7 @@ spec:
 # ...
 ----
 <1> The namespace where the pipelines are to be run.
-<2> The feature gate to be enabled to deploy TTO resources.
+<2> The feature gate to be enabled to deploy SSP resources.
```

Comment:
This is not accurate, please write it as: `The feature gate to be enabled to deploy Tekton resources by SSP operator.`